exonum/src/helpers/mod.rs

```diff
@@ -151,3 +153,28 @@ fn format_log_record(buf: &mut Formatter, record: &Record) -> io::Result<()> {
         writeln!(buf, "{} {} {} {}", time, level, &source_path, record.args())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use regex::Regex;
+
+    use std::time;
+
+    use super::*;
+
+    #[test]
+    fn time_formatting() {
+        // Thu,  1 Jan 1970 00:00:00.000 +0000
+        let r = Regex::new(
```

Comment:
Why compare against regex (and introduce a new dependency), when you can compare against expected stringp?